lib/autosign/validator.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Trivial but this seems unneeded?

Reply:
https://www.mikeperham.com/2018/02/28/ruby-optimization-with-one-magic-comment/